docker-compose.yaml

```
@@ -1,20 +1,9 @@
-
   mongodb:
     image: mongo:latest
-    # no ports! internal only
-
+    ports:
+      - "27017:27017"
   redis:
     image: redis:latest
-    # no ports! internal only
+    ports:
+      - "6379:6379"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)